applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -349,7 +349,33 @@ void le_audio_adv_get(const struct bt_data **adv, size_t *adv_size, bool periodi
 	}
 }
 
-int le_audio_play_pause(void)
+int le_audio_play(void)
+{
+	int ret;
+
+	/* All streams in a broadcast source is in the same state,
+	 * so we can just check the first stream
+	 */
+	if (audio_streams[0].ep == NULL) {
+		LOG_ERR("stream->ep is NULL");
+		return -ECANCELED;
+	}
+
+	if (audio_streams[0].ep->status.state == BT_BAP_EP_STATE_STREAMING) {
```

Comment:
Suggest to use the public `bt_bap_ep_get_info` instead of using internal (private) fields

Reply:
Is that new? I seem to remember we had to use the internal fields since there was no helper function. Will update

Reply:
Will do that when we rewrite the le_audio files into bt_stream.

Reply:
It is fairly new; added to Zephyr 5 months ago